### PR TITLE
enable off chain workers for pendulum node.

### PR DESCRIPTION
**Pull request by cr4pt0**
- enable off-chain workers for pendulum node.
- need redeploy the node. runtime upgrade will not fix issue with workers.

### CODE REVIEW

**Review by TorstenStueber — APPROVED** [review]

File: runtime/foucoco/src/zenlink.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ use sp_runtime::{DispatchError, DispatchResult};
 use sp_std::marker::PhantomData;
 
 use zenlink_protocol::{
-	AssetId, AssetIdConverter, Config as ZenlinkConfig, LocalAssetHandler, PairLpGenerate,
+	AssetId, AssetIdConverter, LocalAssetHandler, PairLpGenerate,
```
Comment (gonzamontiel) [inline runtime/foucoco/src/zenlink.rs:10]: is this related to the pr?

**Review by gonzamontiel — APPROVED** [review]
I approve it, only left an unimportant comment.

File: runtime/foucoco/src/zenlink.rs (hunk 1 above)
Comment (TorstenStueber) [reply runtime/foucoco/src/zenlink.rs:10]: No, but it is an unused import and creates a warning – should have already been removed in a previous commit.
